Data/Sys/GameSettings/GL5.ini

```diff
@@ -17,3 +17,5 @@ EmulationIssues = Slow
 [ActionReplay]
 # Add action replay cheats here.
 
+[Video_Stereoscopy]
+StereoConvergence = 1
```

Comment:
@SMarioMan I see multiple games with extremely low convergence settings such as this. Would support for fractions like `0.25` be useful to you here?